packages/web-react/src/components/Accordion/UncontrolledAccordion.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { UncontrolledAccordionProps } from '../../types';
+import Accordion from './Accordion';
+import { useAccordion } from './useAccordion';
+
+const defaultProps: UncontrolledAccordionProps = {};
+
+const UncontrolledAccordion = (props: UncontrolledAccordionProps) => {
+  const { stayOpen, defaultOpen, ...restProps } = props;
```

Comment:
Just to be unified I would use `defaultOpen` before `stayOpen` as it is defined within its interface. (If you're gonna process it, please do not forget about docs/readme files.)

Reply:
done